layouts/partials/tw/newsletter-full.html

```diff
@@ -0,0 +1,104 @@
+
+<section class=" bg-s-3" data-ga-category="inline">
+  <div
+    class="mx-auto max-w-screen-lg py-6 px-4 md:grid md:grid-cols-10 md:gap-y-3 md:gap-x-5 md:py-24 md:px-0"
+  >
+    <div class="md:col-span-5 md:col-start-1">
+      <h2 class=" max-w-md text-2xl md:text-5xl font-black leading-none  text-black">
+        {{ .hed }}
+      </h2>
+      <div class="article-content my-2 max-w-md text-base">
+        {{ .dek }}
+      </div>
+
+      <form
+        action="{{ site.Param `newsletters-action` }}"
+        method="post"
+        target="_blank"
+        class="font-sans text-black"
+        data-ga-form="{{ $gaForm }}"
+        data-ga-category="{{ $gaCategory }}"
+      >
+        {{ partial $hiddenField . }}
+
+
+        <div class="mb-5 flex flex-wrap justify-start align-baseline">
+          <p class="mr-3 flex-[1_0_45%]">
+            <label class="mt-4 block font-extrabold" for="{{ $idFname }}">
+              First Name
+            </label>
+            <input
+              id="{{ $idFname }}"
+              class="block w-full appearance-none rounded border-2 border-s-4 bg-white py-2 px-4 tracking-wider text-s-8 placeholder-s-4 placeholder-opacity-100 outline-none ring-red-7 focus:ring-2"
+              type="text"
+              name="FNAME"
+              placeholder="William"
+              required
+            />
+          </p>
+
+          <p class="ml-3 flex-[1_0_45%]">
+            <label class="mt-4 block font-extrabold" for="{{ $idLname }}">
+              Last Name
+            </label>
+            <input
+              id="{{ $idLname }}"
+              class="block w-full appearance-none rounded border-2 border-s-4 bg-white py-2 px-4 tracking-wider text-s-8 placeholder-s-4 placeholder-opacity-100 outline-none ring-red-7 focus:ring-2"
+              type="text"
+              name="LNAME"
+              placeholder="Penn"
+              required
+            />
+          </p>
+          <p class="mr-6 flex-[1_0_100%]">
+            <label class="mt-4 block font-extrabold" for="{{ $idEmail }}">
+              Email Address
+            </label>
+            <input
+              id="{{ $idEmail }}"
+              class="block w-full appearance-none rounded border-2 border-s-4 bg-white py-2 px-4 tracking-wider text-s-8 placeholder-s-4 placeholder-opacity-100 outline-none ring-red-7 focus:ring-2"
+              type="email"
+              name="EMAIL"
+              placeholder="billpenn@example.com"
+              required
+            />
+          </p>
+          <button
+            type="submit"
+            class="mt-8 rounded border-2 border-red bg-red px-8 py-2 font-semibold uppercase tracking-widest text-white outline-none ring-red-7 focus:ring-2"
+          >
+            {{ $cta }}
+          </button>
+        </div>
+        <p class=" text-sm text-s-7">{{ .disclaimer }}</p>
+      </form>
+    </div>
+    {{ $name := .image }}
+    {{ $imgOpts := dict
+      "filename" $name
+      "link" .RelPermalink
```

Comment:
I think this isn't passing through? We probably don't want the image to be a link anyway.

Reply:
Removed in https://github.com/spotlightpa/poor-richard/pull/356/commits/06c32bc43222692b3fe2be0d2a63d61de6a1683c